include/coap2/pdu.h

```diff
@@ -246,6 +246,8 @@ typedef int coap_mid_t;
 /** Indicates an invalid message id. */
 #define COAP_INVALID_MID -1
 
+#define COAP_INVALID_TID COAP_INVALID_TID_invalid_use_COAP_INVALID_MID
```

Comment:
I suggest to define COAP_INVALID_TID to COAP_INVALID_MID and mark it as deprecated via a doxygen comment.
And I think this should be in a separate PR.

Reply:
Change to `pdu.h` reverted and code pushed.  Will create a separate PR.